scripts/updateReadmeLineNumbers.py

```diff
@@ -0,0 +1,37 @@
+#Usage: python scripts/updateReadmeLineNumbers.py
+import re
+import sys
+
+edm4hep_yaml_content = ""
+with open("edm4hep.yaml") as edm4hep_yaml:
+  edm4hep_yaml_content = edm4hep_yaml.readlines()
+
+readme_content = ""
+with open("README.md", "r") as readme:
+  readme_content = readme.readlines()
+
+new_readme_content = ""
+
+for readme_line in readme_content:
+  edm4hep_objects = re.findall("\[(.*?)\]\(https:\/\/github.com\/key4hep\/EDM4hep\/blob\/master\/edm4hep\.yaml#L(\d+?)\)", readme_line)
+  if edm4hep_objects:
+    for object_lineNumber_tuple in edm4hep_objects:
+      edm4hep_object = object_lineNumber_tuple[0]
+      original_line_number = object_lineNumber_tuple[1]
+      regex = ".*edm4hep::%s:$"%edm4hep_object
```

Comment:
This doesn't work if someone adds spaces after the colon
```suggestion
      regex = ".*edm4hep::%s: *$"%edm4hep_object
```